lib/main.js

```diff
@@ -70,14 +72,14 @@ module.exports = class Router extends EventEmitter {
         this.app.param('type', params.type);
 
         this.app.post(
-            '/feed/:type',
+            '/feed/:type/:id?',
```

Comment:
Just to doublecheck, including the question mark here and conditionally handling id means that existing routes will continue to work as correct?

Reply:
Yes. Old tests hitting old routes still pass :)